setup.json

```diff
@@ -0,0 +1,13 @@
+{
+  "kolide_server_url": "https://kolide:8412",
```

Comment:
lets change to fleet:8412

Reply:
I updated the hostnames in the most recent push